base/applications/mspaint/registry.h

```diff
@@ -43,7 +43,9 @@ class RegistrySettings
     DWORD ShowStatusBar;
     DWORD ShowPalette;
     DWORD ShowToolBox;
-    DWORD Bar1Flags;
+    DWORD Bar2ID;
+#define BAR2ID_LEFT   0x0000e81c
```

Comment:
OK then I would suggest you add a comment that those values are for Win2k3 mspaint compatibility.

Reply:
Added.